src/qt/locale/bitcoin_de.ts

```diff
@@ -39,7 +43,7 @@ Dieses Produkt enthält Software, die vom OpenSSL-Projekt zur Verwendung im Open
     <message>
         <location filename="../forms/addressbookpage.ui" line="+14"/>
         <source>Address Book</source>
-        <translation>Adressbuch</translation>
+        <translation>Eigene Adresse</translation>
```

Comment:
Are these really your own adresses there? Address Book should be target adresses right?